src/app/pages/ValidatorDetailsPage/BalanceDistributionCard.tsx

```diff
@@ -63,10 +81,28 @@ export const BalanceDistributionCard: FC<BalanceDistributionCardProps> = ({ vali
           data={chartData(t, validator)}
           dataKey="value"
           formatters={{
-            data: (value: number) =>
-              t('common.valueLong', {
-                ...getPreciseNumberFormat(value.toString()),
-              }),
+            data: (value: number) => {
+              const shares = balanceToSharesMap[value.toString()]
```

Comment:
Second commit doesn't always work:
https://pr-1572.oasis-explorer.pages.dev/mainnet/consensus/validators/oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe
```
balanceToSharesMap = {
  "589364.128307357": 418936993010513,
  "177402459.741558506": 126102776648867500
}
value = 177402459.7415585
shares = 0
```
I'd go without it

Reply:
note: if we will need shares use BigInt in map